REFERENCE.md

```diff
@@ -0,0 +1,90 @@
+---
+RFC: <PR number>
+Status: <Proposed>
+---
+
+# Title (Required)
+
+## Abstract (Required)
+
+A few sentences describing the feature.
+
+## Motivation (Required)
+
+What the feature solves and why the existing functionality is not enough.
+
+## Design considerations (Required)
+
+A description of the design constraints and requirements for the proposal. Comparisons with similar features in other projects.
+
+## Specification (Required)
+
+A more detailed description of the feature, including why certain details in the design have been chosen.
+
+### Commands (Optional)
+
+If any new commands are introduced:
+
+1. Command name
+   - **Request**
+   - **Response**
+
+### Authentication and Authorization (Optional)
+
+If there are any changes around authentication and authorization (ACL) access for commands/keyspace/channels.
+
+### Append-only file (Optional)
+
+If there are any changes around the persistence mechanism of every write operation.
+
+### Configuration (Optional)
+
+If there are any configuration changes introduced to enable/disable/modify the behavior of the feature.
+
+### Keyspace notifications (Optional)
+
+If there are any events to be introduced or modified to observe activity around the dataset.
+
+### Cluster mode (Optional)
+
+If there are any changes introduced for cluster mode related activities like cluster bus protocol change, slot migration, etc.
```

Comment:
I thought this was about special considerations around clustermode. I think if there is something about changing clustermode, it would likely not be in this section.